index.bs

```diff
@@ -136,77 +136,156 @@ important to consider ways to mitigate the obvious impacts. For instance:
+to further mitigate the risk of exfiltration.
+
+</div>
+
+Many use cases
+which require location information
+can be adequately served
+with very coarse location data.
+For instance,
+a site which recommends restaurants
+could adequately serve its users
+with city-level location information
+instead of exposing the user's precise location.
+
+See also
+
+* [[DESIGN-PRINCIPLES#do-not-expose-use-of-assistive-tech]]
 
 <h3 class=question id="persistent-origin-specific-state">
   Does this specification introduce new state for an origin that persists
   across browsing sessions?
 </h3>
 
-Allowing an origin to persist data on a user’s device across browsing
-sessions introduces the risk that this state may be used to track a user
-without their knowledge or control, either in a first party or third party
-contexts.  New state persistence mechanisms should not be introduced without
-mitigations to prevent it from being used to track users across domains or
-without control over clearing this state.  And, are there specific caches
-that a user agent should specially consider?
-
-<p class=example>
-Service Worker [[SERVICE-WORKERS]] intercept all requests made by an
-origin, allowing sites to function perfectly even when offline. A
-maliciously-injected service worker, however, would be devastating (as
-documented in [[SERVICE-WORKERS#security-considerations]]).
-They mitigate the risks an [=active network attacker=] or [=XSS=]
-vulnerability present by requiring an encrypted and authenticated
-connection in order to register a service worker.
-</p>
-
-<p class=example>
-Platform-specific DRM implementations might expose origin-specific
-information in order to help identify users and determine whether they
-ought to be granted access to a specific piece of media. These kinds of
-identifiers should be carefully evaluated to determine how abuse can be
-mitigated; identifiers which a user cannot easily change are very
-valuable from a tracking perspective, and protecting the identifiers from
-an active network attacker is an important concern.
-</p>
+There are many existing mechanisms
+origins can use to
+store information about a user—cookies,
+`ETag`,
+`Last Modified`,
+{{localStorage}},
+and
+{{indexedDB}}
+are just a few examples.
+
+Allowing an origin
+to store data
+on a user’s device
+in a way that persists across browsing sessions
+introduces the risk
+that this state may be used
+to track a user
+without their knowledge or control,
+either in [=first-party-site context|first-=] or [=third-party context|third-party=] contexts.
+
+One of the ways
+user agents mitigate the risk
+that client-side storage mechanisms
+will form a persistent identifier
+is by providing users with the ability
+to clear out the data stored by origins.
+New state persistence mechanisms
+should not be introduced
+without mitigations
+to prevent it
+from being used
+to track users
+across domains
+or without control
+over clearing this state.
```

Comment:
Suggested addition:

Further, manually clearing storage is a rare event for users, since it can break sessions and be a tedious step while browsing.  Spec authors should therefor consider ways new features can be privacy-preserving without full storage clearing, such as reducing the uniqueness of values, rotating values, or otherwise, making features no more identifying than is needed.

Reply:
Added in 04e2674.